release_notes_generator/generator.py

```diff
@@ -72,15 +75,13 @@ def generate(self) -> Optional[str]:
 
         @return: The generated release notes as a string, or None if the repository could not be found.
         """
+        # get the repository
         repo = self._safe_call(self.github_instance.get_repo)(ActionInputs.get_github_repository())
         if repo is None:
             return None
 
-        rls = self._safe_call(repo.get_latest_release)()
-        if rls is None:
-            logger.info("Latest release not found for %s. 1st release for repository!", repo.full_name)
-        else:
-            logger.debug("RLS created_at: %s, published_at: %s", rls.created_at, rls.published_at)
+        # get the latest release
+        rls = self.get_latest_release(repo)
```

Comment:
I would suggest adding a type hint here.

Reply:
Fixed in commit 3bd9b0f89377c110b90a3c003373aa1e44cc7930.